packages/itwinui-css/src/searchbox/searchbox.scss

```diff
@@ -29,10 +29,3 @@
     padding-inline: 0; // To center iui-searchbox-open-button inline in collapsed state
   }
 }
-
-.iui-search-icon {
```

Comment:
Is this a breaking change?

It looks like we only use `iui-search-icon` and `iui-search-input` for padding adjustments, so I'm ok with pushing it in a minor release.

All instances of these classes should still be removed though.

Reply:
Although `iui-search-icon` and `iui-search-input` are no longer used for any styling at the moment, would it be alright to still keep it in our itwinui-css API? That way if we need these classes in the future for some custom styling, we can re-use them.

If by iTwinUI v4 we don't have any use of these classes, then we can remove them in v4 as a breaking change.

Reply:
Sure, but lets maybe add empty rules with a comment?

```scss
// noop
.iui-search-icon {}
.iui-search-input {}
```

Reply:
Sounds good, done (https://github.com/iTwin/iTwinUI/pull/1881/commits/623180c5a8f51e8542f75ba5dc814473ec891e63).